### DISCUSSION TITLE
How to open an URL/default browser from a tauri::command?

### DISCUSSION THREAD

**Discussion by matthiasbeyer** [tauri-apps/tauri#5857] [Q&A] [answered]
Hi,

I want to open the browser (a URL) from a `tauri::command`, but the `shell::open` function needs some shell context that I don't know how to provide.

Unfortunately there's also no example for this exact case... so I figured I have to ask here...

**Comment by FabianLars** [comment, marked as answer]
You could use this crate <https://crates.io/crates/open> it's the same thing that powers shell::open (the latter is btw only really implemented for the JS api, so we prob should make it private with these weird args)
